README.md

```diff
@@ -106,11 +106,15 @@ This dbt package is dependent on the following dbt packages. Please be aware tha
 > IMPORTANT: If you have any of these dependent packages in your own `packages.yml` file, we highly recommend that you remove them from your root `packages.yml` to avoid package version conflicts.
 ```yml
 packages:
-    - package: fivetran/fivetran_utils
-      version: [">=0.3.0", "<0.4.0"]
+  - package: fivetran/fivetran_utils
+    version: [">=0.3.0", "<0.4.0"]
+
+  - package: dbt-labs/dbt_utils
+    version: [">=0.8.0", "<0.9.0"]
+
+  - package: dbt-labs/spark_utils
```

Comment:
Do we need to add `spark_utils` package dependencies to README in our own databricks compatible packages? Or is this specific to this issue and this package?

Reply:
This was actually I think an oversight when we did the README updates. This spark utils dependency should be included in all READMEs that have databricks compat!

Reply:
Ahh makes sense!